File: Python3/saldo.py
"""Problema: Fazer um algoritmo que leia o saldo inicial de cliente do banco
e leia também um cheque que entrou e ANALISE se o cheque poderá ser descontado
ou não , já que este cliente não possui limite. Se o cheque não poderá
ser descontado, mostre essa informação, caso contrário, desconte o
cheque e informe o saldo na tela."""

valor_saldo = int(input("Digite o valor do saldo inicial: "))
cheque_entrou = int(input("Digite o valor do cheque para entrar: "))
if cheque_entrou <= valor_saldo:
     valor_saldo = valor_saldo - cheque_entrou
     print("O seu novo saldo: %f" %(valor_saldo))
else:
     print("Atencao, este cheque nao pode ser descontado: ")
     print("O seu saldo continua: %f" %(valor_saldo))

File: C/saldo.c

/* 1.	Problema: Um cliente de um banco tem um saldo positivo de R$ 500,00. 
Fazer um algoritmo que leia um cheque que entrou e calcule o saldo, 
mostrando (escrevendo) o saldo na tela.
*/

#include <stdio.h>
int main()
{
   float valor_saldo, cheque_entrou;
   valor_saldo = 500;
   printf("Digite o valor do cheque que entrou: ");
   scanf("%f",&cheque_entrou);
   valor_saldo = valor_saldo - cheque_entrou;
   printf("O saldo Atual: %f\n",valor_saldo);         
return 0;
}
